README.md

```diff
@@ -103,16 +88,6 @@ After you're finished please remove all the comments and instructions!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- GETTING STARTED -->
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly remove the third author from your readme file since she never contributed to this Pull Request. This will make your readme file more professional.